SONAR-netCDF4/docs/tableBeamGroup1.adoc

```diff
@@ -143,4 +145,12 @@
 |int receive_transducer_index(beam) |MA |Receiving or monostatic transducer index associated with the given beam
 |:valid_min = "0" | |
 |:long_name = "Receive transducer index" | |
-|=========================================================================================================================================================================================================================================================================================================================================================
+| | |
+|int active_MRU(ping_time)|MA |Indicate the index of the MRU sensor used at the time of the ping to compute the platform attitude.
+|:valid_min = "0" | |
+|:long_name = "Active MRU sensor index"| |
+| | |
+|int active_position_sensor(ping_time)|MA |Indicate the index of the position sensor used at the time of the ping to compute the platform position.
+|:valid_min = "0" | |
+|:long_name = "Active position sensor index"| |
+ =========================================================================================================================================================================================================================================================================================================================================================
```

Comment:
The end of table marker needs a | before the ='s (generates an error when running through asciidoctor otherwise).

Reply:
My mistake, sorry